samples/sensor/mcux_lpcmp/sample.yaml

```diff
@@ -0,0 +1,24 @@
+sample:
+  description: Demonstration of the NXP MCUX LPCMP driver
+  name: NXP MCUX LPCMP sample
+common:
+  platform_allow:
+    - frdm_mcxn947/mcxn947/cpu0
+  integration_platforms:
+    - frdm_mcxn947/mcxn947/cpu0
+  tags:
+    - drivers
+    - sensor
+tests:
+  sample.sensor.mcux_lpcmp:
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Adjust LPCMP positive input voltage by turning the external voltage supply."
+        - "LPCMP input [a-z]* threshold"
+  sample.sensor.mcux_lpcmp.no_trigger:
```

Comment:
How would this sample work without a trigger? The driver does not seem to support polled operation, since no `channel_get` implementation is provided